signalfx/provider.go

```diff
@@ -153,27 +172,38 @@ func signalfxConfigure(data *schema.ResourceData) (interface{}, error) {
 		config.CustomAppURL = customAppURL.(string)
 	}
 
-	var netTransport = logging.NewTransport("SignalFx", &http.Transport{
-		Proxy: http.ProxyFromEnvironment,
-		DialContext: (&net.Dialer{
-			Timeout: 5 * time.Second,
-		}).DialContext,
-		TLSHandshakeTimeout: 5 * time.Second,
-		MaxIdleConns:        100,
-		MaxIdleConnsPerHost: 100,
-	})
+	pooledTransport := cleanhttp.DefaultPooledTransport()
+	pooledTransport.DialContext = (&net.Dialer{
+		Timeout: 5 * time.Second,
+	}).DialContext
+	pooledTransport.TLSHandshakeTimeout = 5 * time.Second
+	pooledTransport.MaxIdleConnsPerHost = 100
+
```

Comment:
If we are migrating to `cleanhttp` , let's set `MaxIdleConns` as before

Reply:
Reverted back to using `net/http` and included `MaxIdleConns` again